### PR TITLE
State the estimated required balance to stake - Issue #154

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
PR to solve issue #154 

* Added the suggested text by @jamescowens 
* Rearranged some text and removed redundant test,
* Fixed and linked to exchange.htm as a singular way to find exchanges without pointing to other guide pages which could confuse users.
* Removed a unavailable YouTube video guide while I was at it.
*Added a new partial to prevent these changes also affecting the pool guide where most of this info is unnecessary

Review away~~~~~~

### CODE REVIEW

File: source/layouts/partials/_solo_boinc_intro.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="row">
+            <div class="col-xs-12">
+                <h4 class="text-bold">Install BOINC</h4>
+            </div>
+        </div>
+
+        <!--Step 1-->
+        <div class="row">
+            <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                <p>Navigate to the <a href="https://boinc.berkeley.edu">BOINC website</a>, download and install the appropriate BOINC client on each of the computers you wish to crunch BOINC work upon. If you're not crunching projects which require <a href="https://www.virtualbox.org/wiki/Downloadsl">VirtualBox</a> you don't need to install BOINC+VirtualBox.</p>
```
Comment (barton2526) [inline source/layouts/partials/_solo_boinc_intro.erb]: link should be `https://www.virtualbox.org/wiki/Downloads` without the "l" at the end.

File: source/layouts/partials/_solo_boinc_intro.erb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                <a href="https://boinc.berkeley.edu/download.php"><button class="btn btn-round grcbtn">BOINC client downloads</button></a>
+            </div>
+            <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                Here's some BOINC installation video tutorials:
+                    <ul>
+                        <li><a href="https://www.youtube.com/watch?v=SiQ0Ro_bCZY">Pi my life up: Setting up BOINC on a Raspberry PI</a></li>
```
Comment (barton2526) [inline source/layouts/partials/_solo_boinc_intro.erb]: suggest lowercase i in "Pi".
"Pi my life up: Setting up BOINC on a Raspberry Pi"

File: source/assets/css/gridcoin.css [hunk 3]
```diff
@@ -3,7 +3,7 @@ body {
     overflow-x: hidden;
     margin-top: 0px;
     padding-top: 0px;
-    background-color: rgb(44, 44, 44);
+    background-color: #1A2632;
```
[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply source/assets/css/gridcoin.css]: Documented as 'Fix for margin misscolored bits', Basically I set the global background colour to be the same as the main content areas, this is so margin and padding both have the same colours.

Essentially it fixes a colour miss-match in pages where the main content need both a margin and padding to come below the navbar as is the case in exchange.htm

[the remark this answers is not in the document]
Comment (ShmoogleOsukami) [reply source/assets/css/gridcoin.css]: That is, I'm exclusively using bootstrap margins and paddings which in the case of exchange.htm is mt-5 pt-5, Without this colour change padding would be #1A2632 and the margin would be rgb(44,44,44)

**Pull request merged**